evap/templates/base.html

```diff
@@ -92,6 +92,15 @@
                             </li>
                         {% endif %}
 
+                        {% if user.is_reviewer %}
```

Comment:
i thought it would be nice to have that for staff users as well if they just want to review sth.